cics/Z390SEQ.MLC

```diff
@@ -1,7 +1,20 @@
 *********************************************************************
-* Copyright 2006 Automated Software Tools Corporation               *
-* This source code is part of z390 assembler/emulator package       *
-* The z390 package is distributed under GNU general public license  *
+* z390 - Mainframe assembler emulator and run-time engine
+* Copyright (C) 2021 z390 Assembler LLC
+*
+* This file is part of z390.
+*
+* z390 is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+* z390 is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License 
+* along with this program; if not, see https://www.gnu.org/licenses.
 * Author - Don Higgins                                              *
 *********************************************************************
```

Comment:
I've noticed the following throughout, but only now decided to comment.

The "Author - Don Higgins" is not part of the copyright statement, is it?  Shouldn't there be a line of asterisks above this line, separating it from the copyright?

Reply:
OK.. so I have done a bulk update to add a line between the last copyright line and the Author line. It will provide what you have requested for most cases. Could be a couple of missed items or rough edges.. but provides the above.